### DISCUSSION TITLE
What's the best fork?

### DISCUSSION THREAD

**Discussion by dogweather** [jekyll/jekyll#9486] [General]
The last release was cut in January. Although PR's and commits are getting merged.

https://repocheck.com/#jekyll%2Fjekyll

 Can anyone recommend a fork?

**Comment by mattr-** [comment]
Why do you think that a fork is the answer?

**Reply by dogweather** [reply]
I'd like to benefit from the work done over the past 11 months in a stable way.

Would you consider releasing a new version?

**Reply by mattr-** [reply]
Most of the work done has been documentation and dependency updates. Part of the reason we haven't released yet is that there's only been one minor enhancement, specifically to allow marking specific highlighted lines via Liquid, and only a handful of bug fixes.  

There's simply not enough material to release that would normally justify making a release right now. [Our changelog](https://github.com/jekyll/jekyll/blob/master/History.markdown) has all the details of what's unreleased. In particular, if there's an optimization that you think will be super beneficial to you based on your site design I'd like to know and I'll move cutting a release higher on my priority list.

**Reply by dogweather** [reply]
Would you consider delegating this task? I think it's a nice agile best practice to create regular releases.

**Reply by mattr-** [reply]
Unless I've misunderstood what you're asking, there's no one to delegate that task to. I'm currently the only active maintainer as other folks have stepped away for the time being.

**Reply by dogweather** [reply]
I'd be happy to do it. I saw that you have a `release.yml` workflow. I can do a dry run to a throwaway gem name in a fork.

**Reply by mattr-** [reply]
There are a few things in flight that I want to get merged before I'd consider doing an actual release. Once those are done, 4.4 will likely land. Best guess based on current availability is around the new year.

**Comment by shushugah** [comment]
@mattr thank you for holding down the fort. I saw an extensive discussion about jekyll 5 in #9156. For me specifically, integrating multu-language support natively would make jekyll a much better ssg choice. I am one of the maintainers of [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin) but it would be nice to merge the project somehow.

**Reply by mattr** [reply]
You're welcome, but honestly @mattr- (👋) deserves all the credit 😄

**Reply by mattr-** [reply]
I feel like you deserve some credit considering how many issue mentions you get that are meant for me. 😅

**Reply by mattr-** [reply]
@shushugah I'll look at #9156 again after 4.4 is released. Appreciate the reminder.

**Comment by jhoney124** [comment]
"Maintainer,
I believe you deserve acknowledgment, given the number of issue mentions directed towards me that you handle."

**Reply by jhoney124** [reply]
Maintainer,
I believe you deserve acknowledgment, given the number of issue mentions directed towards me that you handle.
